src/templates/healthcert/parsers/pdtHealthCertV2/generateMultiQrSection.tsx

```diff
@@ -45,6 +46,9 @@ export const generateMultiQrSection = (
     });
   }
 
+  const logoWidth = 130;
+  const logoHeight = logoWidth / 5;
```

Comment:
https://github.com/Notarise-gov-sg/healthcert-renderer/pull/125/files?short_path=095592d#r778645199

Reply:
it is specified in the UI wireframe i believe, @witxhhaven